charts/commondata-backend/values.yaml

```diff
@@ -42,6 +42,24 @@ database_password: #
 redis_host:
 
 ingress:
+  enabled: true
+  hosts:
+    - sample.com
+  # The ingress path. Useful to host metabase on a subpath, such as `/metabase`.
+  path: /
+  labels:
+    # Used to add custom labels to the Ingress
+    # Useful if for example you have multiple Ingress controllers and want your Ingress controllers to bind to specific Ingresses
+    application: foresight-frontend
```

Comment:
This says application: foresight-frontend but this is commondata-backend/values.yaml. Can we remove this if it doesn't cause errors